web/docs/Backend Starters/Decoupled Drupal/creating-new-project.md

```diff
@@ -1,25 +1,167 @@
+
+```
+
+git branch -D master
+
+git checkout -b master
+
+git push origin master --force
+
+```
+
+:::Note
+Make sure code has synced and all active workflows are complete in the UI.
+Click on the **Visit Development Site** button to Install via the UI—selecting either the Pantheon Decoupled Profile, or Pantheon Decoupled Umami Demo profiles.
+:::
+
+## Installing using Build Tools
+
+### Prerequisites
 
 - Composer (required for CMS backends): [Install Globally](https://getcomposer.org/download/)
 - [Generate machine token](https://pantheon.io/docs/machine-tokens#create-a-machine-token) & [Authenticate into Terminus](https://pantheon.io/docs/machine-tokens#authenticate-into-terminus)
 - [Install Terminus](https://pantheon.io/docs/terminus/install) (3.0.0 above required)
 - Also install the following plugins:
   - `terminus self:plugin:install terminus-build-tools-plugin`
-  - `terminus self:plugin:install terminus-power-tools`
+  - `terminus self:plugin:install terminus-power-tools` [Known Issues](#known-issues)
   - `terminus self:plugin:install terminus-secrets-plugin`
   - Reload the terminus plugins: `terminus self:plugin:reload`
   - Clear cache for composer: `composer clear-cache`
   - Validate that the required plugins are installed: `terminus self:plugin:list`
 - Create [Github Personal access tokens](https://github.com/settings/tokens)
-- Create [CircleCI Personal API Tokens](https://app.circleci.com/settings/user/tokens)
+- [For other providers](#additional-tokens)
+- Build Tools should prompt the user for the appropriate tokens during [Installation](#installation)
 
-## Installation
+#### Known Issues
+
+- If you encounter errors during the [Installation](#installation) process below. You may remove the terminus-power-tools plugin and running the install process again.
+
+#### Additional tokens
+
+- Create [CircleCI Personal API Tokens](https://app.circleci.com/settings/user/tokens)
+- Create [Bitbucket app password](https://bitbucket.org/account/settings/app-passwords/) for your [username](https://bitbucket.org/account/settings/username/change/)
+- Create [GitLab Personal API Tokens](https://gitlab.com/-/profile/personal_access_tokens)
+
+### Installation
+
+- For all steps below
+
+  - Replace `{PROJECT_NAME}` with your project name - for example `decoupled-drupal`.
+
+  - Replace `'{My Team Name}'` with your team name - for example `My Agency`. This can also be omitted.
+
+1.  Run the `terminus build:project:create` as follows for Github with CircleCI:
+
+```
+terminus build:project:create \
+  --team='{My Team Name}' \
+  --template-repository="git@github.com:pantheon-systems/decoupled-drupal-composer-managed.git" pantheon-upstreams/decoupled-drupal-composer-managed \
+  --visibility private {PROJECT_NAME} \
+  --profile="pantheon_decoupled_profile" \
+  --stability=dev
+```
+
+2. Run the `terminus build:project:create` as follows for Github with Github Actions:
```

Comment:
I had previously suggested:

> Rather than repeating this four times, could we instead use one generic example and then include the 'Additional options` section from the existing documentation?

And I still think we should make that change.